### ISSUE TITLE
Clear output and clear console should announce the content is cleared

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52399]
Environment Details:
VSCode Version : 1.23.1
OS Version : Win10
Mac OS Version:HighSierra10.13.5 

Repro Steps:
1. Launch VS Code while NVDA/VoiceOver is ON.
2. Go to View menu and Select Output(Ctrl+Shift+U).
3. Tab till Clear Output button.

Actual:
NVDA does not announce any information when user clicks on Clear Output button. Visually Output pane gets cleared but screenreader does not announce anything. Issue also found with VoiceOver and Jaws.

Expected:
ScreenReader should provide some information when user clicks on Clear Output button that Output pane is blank now or Data has been cleared something like this.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

Attachment for reference:
[A11y VSCode Output Clear Output button.pptx](https://github.com/Microsoft/vscode/files/2118146/A11y.VSCode.Output.Clear.Output.button.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
I decided to announce that the output and console are cleared only if a manual action causes this.
So for example if a new debug session is started debug console is cleared automatically but I do not announce this as I feel it is spammy. User will see it is cleared once the navigates to the console
